edit-post/README.md

```diff
@@ -57,7 +57,7 @@ The text within the component appears as the menu item label.
 ```jsx
 <PluginMoreMenuItem
 	name="my-plugin"
-	icon={ <wp.components.DashIcon name="yes" /> }
+	icon={ <wp.components.DashIcon icon="yes" /> }
```

Comment:
This should be:
```js
icon="yes"
```

Reply:
I guess I fixed that while you were code reviewing :)